Lib/traceback.py

```diff
@@ -1106,6 +1106,11 @@ def __init__(self, exc_type, exc_value, exc_traceback, *, limit=None,
             suggestion = _compute_suggestion_error(exc_value, exc_traceback, wrong_name)
             if suggestion:
                 self._str += f". Did you mean: '{suggestion}'?"
+        elif exc_type and issubclass(exc_type, ModuleNotFoundError) and \
+                sys.flags.no_site and \
+                getattr(exc_value, "name", None) not in sys.builtin_module_names:
+            self._str += (". Site initialization is disabled, did you forget to "
+                + "add the site-package directory to sys.path?")
```

Comment:
It's `site-packages`, not `site-package`.
```suggestion
                + "add the site-packages directory to sys.path?")
```